ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -89,6 +89,8 @@ cd dev-scripts
 
 cp /root/pull-secret /root/dev-scripts/pull_secret.json
 
+# provoking a failure in the bootstrap
+echo "export ROOT_DISK=/dev/sdz" >> /root/dev-scripts/config_root.sh
```

Comment:
This didn't seem to work, e2e-metal-ipi passed

Reply:
hmm, looks like we need to update config_example.sh (will do this), it should be ROOT_DISK_NAME=...

Reply:
https://github.com/openshift-metal3/dev-scripts/pull/1181